spec.html

```diff
@@ -34559,8 +34559,23 @@ <h1>%TypedArray%.prototype.entries ( )</h1>
 
       <emu-clause id="sec-%typedarray%.prototype.every">
         <h1>%TypedArray%.prototype.every ( _callbackfn_ [ , _thisArg_ ] )</h1>
-        <p>%TypedArray%`.prototype.every` is a distinct function that implements the same algorithm as `Array.prototype.every` as defined in <emu-xref href="#sec-array.prototype.every"></emu-xref> except that the *this* value's [[ArrayLength]] internal slot is accessed in place of performing a [[Get]] of *"length"*. The implementation of the algorithm may be optimized with the knowledge that the *this* value is an object that has a fixed length and whose integer-indexed properties are not sparse. However, such optimization must not introduce any observable changes in the specified behaviour of the algorithm and must take into account the possibility that calls to _callbackfn_ may cause the *this* value to become detached.</p>
-        <p>This function is not generic. ValidateTypedArray is applied to the *this* value prior to evaluating the algorithm. If its result is an abrupt completion that exception is thrown instead of evaluating the algorithm.</p>
+        <p>The interpretation and use of the arguments of %TypedArray%`.prototype.every` are the same as for `Array.prototype.every` as defined in <emu-xref href="#sec-array.prototype.every"></emu-xref>.</p>
+        <p>When the `every` method is called with one or two arguments, the following steps are taken:</p>
```

Comment:
I prefer phrasing like

```suggestion
        <p>The `every` method performs the following steps when called:</p>
```

since it's true regardless of how many arguments are given. I see that you've copied this phrasing from the `Array.prototype` methods. We should change those, too. Not sure whether we should do that before or after this PR. Either way, I don't think we should land the phrasing you have here.

Reply:
> Either way, I don't think we should land the phrasing you have here.

I don't follow. What you're describing would constitute a wide-reaching editorial PR so it doesn't seem like it should hold up this one?

Reply:
It's not wide-reaching; we use the phrasing I suggest extensively throughout the spec. What I'm asking is that we change it in this PR because it's better to have the better phrasing even if we are inconsistent with the phrasing used in Array until we fix it there.

Reply:
@michaelficarra I would strongly prefer to match the Array methods for now and change them all later. This isn't the appropriate place to do that kind of revision, even just scoped to the parts touched by this PR.

Reply:
`When [^\.\n]+ is called with` occurs 210 times in the spec.

Reply:
@rkirsling Did you look at those occurrences? The array methods are unique in the way they number the arguments. If you want to want name the arguments here, that would be fine too:

```suggestion
        <p>When the `every` method is called with argument _callbackfn_ and optional argument _thisArg_, the following steps are taken:</p>
```

@bakkot I don't know why we would prefer to introduce imprecise wording, even temporarily. We would match basically everything but the Array method phrasing, which is fine by me.

Reply:
I see; I thought you were taking issue with listing off the arguments in general.

Reply:
@michaelficarra Using identical wording to the corresponding sections in Array means that readers don't need to consider why the wording is different, which is good.

Also, the decision of how to phrase this is an editorial one which unrelated to the focus of the PR. I would in general prefer not to pull in unrelated changes; we can fix this up *later* as part of fixing up [the rest](https://github.com/tc39/ecma262/pull/2294#issuecomment-773000412).

Reply:
Discussed this in the editor call. We'll try to fix all these up at some future time (opened https://github.com/tc39/ecma262/issues/2304 to track) and land this as-is for now.